app/views/shared/_top_navbar.html.erb

```diff
@@ -5,8 +5,8 @@
       <span class="fa fa-bars"></span>
     </button>
     <%= link_to 'http://library.stanford.edu' do %>
```

Comment:
Taking a quick look over the app there are a lot of http references.  Going to create a ticket about updating all of them holistically.